docker/packages_license_check_exclude.json

```diff
@@ -54,6 +54,10 @@
         "incydr": {
             "docker_images": ["py42"],
             "notes": "The package has MIT license, however it is not formatted properly in Pypi"
+        },
+        "zipencrypt": {
+            "docker_images": ["py3-tools"],
+            "notes": "The package has MIT license, however it is not formatted properly in Pypi"
```

Comment:
[this is the info from pypi](https://pypi.org/pypi/zipencrypt/json) the license key is empty but in the [repo](https://github.com/devthat/zipencrypt/blob/master/LICENSE.txt) we can see the license is MIT